index.html

```diff
@@ -1,28 +1,104 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
+  <head>
+    <meta charset="UTF-8" />
     <title>Grid Render</title>
     <style type="text/css">
-        html,
-        body {
-            margin: 0;
-            height: 100%;
-            background-color: #017DBD;
-            overflow: hidden;
+      html,
+      body {
+        margin: 0;
+        height: 100%;
+        background-color: #017dbd;
+        overflow: hidden;
+        font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+      }
+
+      .control-panel {
+        background-color: white;
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding: 1rem;
+        border-bottom:5px ridge #020D78;
+
+        .controls {
+          display: flex;
+          justify-content: center;
+          margin-bottom: 1rem;
+
+          .preference {
+            font-size: 1rem;
+            display: flex;
+            flex-direction: column;
+            margin: 0 1rem;
+
+            input {
+                font-size: 1rem;
+                border-top: none;
+                border-left: none;
+                border-right: none;
+                border-bottom: 2px solid #020D78;
+            }
+          }
         }
+
+        .applybtn {
+            border: none;
+            height: 2rem;
+            width: 8rem;
+            background-color: #017dbd;
+            color: white;
+            font-size: 1rem;
+            font-weight: 600;
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+
+            box-shadow: 5px 5px #020D78;
+        }
+        .applybtn:active {
+            translate: 5px 5px;
+            box-shadow: none;
+        }
+      }
     </style>
-</head>
-<body>
+  </head>
+  <body>
+    <div class="hexplosion">
+      <div class="control-panel">
+        <div class="controls">
+            <div class="preference">
+              <label>Hexagon size</label>
+              <input id="radius" placeholder="Hexagon size" />
```

Comment:
Could we add default values to the inputs, when it loads they are empty but it obviously still works.